tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -67,11 +69,65 @@ def _generate_dummy_dataset(self) -> torch.utils.data.DataLoader:
         return torch.utils.data.DataLoader(dataset, batch_size=1)
 
 
+class Model_DDP(pl.LightningModule):
```

Comment:
A nit, but please use the [CapWords convention](https://www.python.org/dev/peps/pep-0008/#class-names). (c.f., https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#general-rules)


```suggestion
class ModelDDP(pl.LightningModule):
```